openupgrade_framework/odoo_patch/odoo/modules/graph.py

```diff
@@ -1,12 +1,16 @@
 # Copyright Odoo Community Association (OCA)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import os
+
 import odoo
 from odoo.modules.graph import Graph
 
 
 def update_from_db(self, cr):
     """Prevent reloading of demo data from the new version on major upgrade"""
     Graph.update_from_db._original_method(self, cr)
+    if os.environ.get("OPENUPGRADE_USE_DEMO", "") == "ok":
+        return
```

Comment:
It will only executed in the case of production databases generated with demo data and then the demo data was customized. Thus, the script will transform that database to a not demo one.